aws/offsite/lambda/verify-database-copy/index.js

```diff
@@ -0,0 +1,114 @@
+console.log('Loading function');
+
+const AWS = require('aws-sdk');
+const mysqlPromise = require('promise-mysql');
+const Slack = require('slack-node');
+const promisify = require('util-promisify');
+const setTimeoutPromise = promisify(setTimeout);
+
+const SLACK_WEBHOOK_URL = `https://hooks.slack.com/services/${process.env.SLACK_WEBHOOK_URL}`;
+const SLACK_CHANNEL = process.env.SLACK_CHANNEL;
+
+const DB_INSTANCE_IDENTIFIER = process.env.DB_INSTANCE_IDENTIFIER;
+const DB_HOST = process.env.DB_HOST;
+const DB_NAME = process.env.DB_NAME;
+const DB_USER = process.env.DB_USER;
+const DB_PASSWORD = process.env.DB_PASSWORD;
+
+var status_message = '';
+
+
+var postStatusToSlack = function postStatusToSlack(message) {
+    console.log('Posting status to Slack');
+    var slack = new Slack();
+    slack.setWebhook(SLACK_WEBHOOK_URL);
+    slack.webhook({
+            channel: SLACK_CHANNEL,
+            text: message
+        },
+        function (error, response) {
```

Comment:
this looks like it could be `promisify`'ed, since the last argument is a node-style callback. however what you have here is simple enough, so this seems optional